test/Assertion/TestSubclasses/AssertionAggregate.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/**
+ * @see       https://github.com/laminas/laminas-permissions-acl for the canonical source repository
+ * @copyright https://github.com/laminas/laminas-permissions-acl/blob/master/COPYRIGHT.md
+ * @license   https://github.com/laminas/laminas-permissions-acl/blob/master/LICENSE.md New BSD License
+ */
+namespace LaminasTest\Permissions\Acl\Assertion\TestSubclasses;
+
+final class AssertionAggregate extends \Laminas\Permissions\Acl\Assertion\AssertionAggregate
+{
+    /**
+     *
+     * @return array
+     */
```

Comment:
According to https://docs.laminas.dev/laminas-coding-standard/v2/coding-style-guide/#9-commenting-and-docblocks, unnecessary comments should not be introduced.

```suggestion
```